app/atomic_api.py

```diff
@@ -0,0 +1,28 @@
+import json
+import logging
+from aiohttp import web
+from aiohttp_jinja2 import template
```

Comment:
nit: https://www.python.org/dev/peps/pep-0008/#imports

Imports should be ordered lexicographically within the following groups (newlines between each group):

* Standard library
* Third party
* Internal packages

I think if you ask PyCharm to optimize your imports it'll order it for you in that manner (and remove any that aren't being used).